NEWS.rst

```diff
@@ -2,6 +2,41 @@
 Release Notes
 =============
 
+snakeoil 0.10.0 (2022-09-18)
+----------------------------
+
+This release has various breaking changes, with various deprecated stuff
+removed. We are planing to remove more stuff in the future, so please be aware.
+All removed stuff will be listed in the release notes, and a better replacement
+will be provided.
+
+This is also the first release to use ``flit`` for packaging, which simplifies
+the build a lot. We include a makefile for convenience of running various build
+commands.
+
+Also note that ``dist.distutils_extensions`` is now deprecated, and will be
```

Comment:
I would explicitly include this as a bullet point as well, because then it's harder to miss.